src/ert/config/queue_config.py

```diff
@@ -112,6 +112,28 @@ def from_dict(cls, config_dict: ConfigDict) -> QueueConfig:
             ):
                 submit_sleep = float(values[0])
 
+        def _is_simulator(fm_name: str):
+            return any(item in fm_name.lower() for item in ["rms", "flow", "eclipse"])
```

Comment:
The forward models for Eclipse are usually called ECLIPSE100 and ECLIPSE300. Do we want to do `startswith` instead?

Reply:
It might be the case for many FLOW runs as well, that they only start with FLOW, but then it is probably a user-defined fm, f.ekx. "FLOW_DAILY".

Reply:
You are right this is a bit ambiguous, it might be better to have a list of known simulators and match on the exact name ["RMS", "FLOW", "ECLIPSE100", "ECLIPSE300"] this way we don't risk adding tags that for jobs that are simulator related